llvm/runtimes/CMakeLists.txt

```diff
@@ -503,17 +513,9 @@ if(build_runtimes)
   endif()
 
   if("openmp" IN_LIST LLVM_ENABLE_RUNTIMES)
-    if (${LLVM_TOOL_FLANG_BUILD})
-      message(STATUS "Configuring build of omp_lib.mod and omp_lib_kinds.mod via flang-new")
-      set(LIBOMP_FORTRAN_MODULES_COMPILER "${CMAKE_BINARY_DIR}/bin/flang-new")
-      set(LIBOMP_MODULES_INSTALL_PATH "${CMAKE_INSTALL_INCLUDEDIR}/flang")
-      # TODO: This is a workaround until flang becomes a first-class project
-      # in llvm/CMakeList.txt.  Until then, this line ensures that flang-new is
-      # built before "openmp" is built as a runtime project.  Besides "flang-new"
-      # to build the compiler, we also need to add "module_files" to make sure
-      # that all .mod files are also properly build.
-      list(APPEND extra_deps "flang-new" "module_files")
-    endif()
+    if (LLVM_TOOL_FLANG_BUILD AND "FortranRuntime" IN_LIST LLVM_ENABLE_PROJECTS)
+      set(enable_fortran ENABLE_FORTRAN)
+    endif ()
```

Comment:
Can't we simply require that `fortran-runtime` be passed in `LLVM_ENABLE_RUNTIMES`? Again, I don't see a `ENABLE_CPLUSPLUS` here so I'm suspicious.

Reply:
`LLVM_ENABLE_PROJECTS` is a mistake by me. This entire change is not needed anymore since the reason for it is are `*.mod` files that in a previous version of this had been created in `FortranRuntime`, but now leave in Flang because it would require a relatively long tail of additional changes.

I am going to remove this change from this PR.